opensearch/output.tf

```diff
@@ -49,6 +49,7 @@ locals {
 
 output "configuration" {
   description = "Cloudinit compatible opensearch configurations"
+  sensitive   = true
```

Comment:
Very good initiative. I would put it on the inputs that contain certs or passwords too.